bulkimport/src/main/java/com/microsoft/azure/documentdb/bulkimport/InsertMetrics.java

```diff
@@ -26,7 +26,8 @@
 
 import com.google.common.base.Preconditions;
 
-class InsertMetrics {
+// TODO: Rename to OperationMetrics and rename 'numberOfDocumentsInserted' to 'numberOfDocumentsOperated'
+public class InsertMetrics {
```

Comment:
why does this have to be public?

Reply:
Since it has to be accessible to the bulk update package. We're going to be adding delete functionality next too, so we need to separate the files into packages.